halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -96,14 +97,17 @@ public void accept(ConfigProblemSetBuilder psBuilder, Validator v) {
   public void makeBootstrappingAccount(ArtifactSourcesConfig artifactSourcesConfig) {
     super.makeBootstrappingAccount(artifactSourcesConfig);
 
+    DeploymentConfiguration deploymentConfiguration = parentOfType(DeploymentConfiguration.class);
+    String location = StringUtils.isEmpty(deploymentConfiguration.getDeploymentEnvironment().getLocation()) ? "spinnaker" : deploymentConfiguration.getDeploymentEnvironment().getLocation();
+
```

Comment:
Isn't this the one that @johnrengelman said would mess up the YAML if it didn't stay static?